Client/Pages/Transactions.razor

```diff
@@ -0,0 +1,151 @@
+                            <td>@t.DestinationWalletId</td>
+                            <td>@t.Amount</td>
+                            <td>@t.Date</td>
+                        </tr>
+                    }
+                </table>
+            }
+            else
+            {
+                <h6 class="card-subtitle mb-2 text-muted">You don't have any made transaction...</h6>
+            }
+            <br />
+
+            <h5 class="card-title">Transfers Recived :</h5>
+            @if (RecivedTransactions.Count > 0)
+            {
+                <table style="width:100%">
+                    <tr>
+                        <th>SourceWalletId</th>
+                        <th>DestinationWalletId</th>
+                        <th>Amount</th>
+                        <th>Date</th>
+                    </tr>
+                    @foreach (var t in RecivedTransactions)
+                    {
+                        <tr>
+                            <td>@t.SourceWalletId</td>
+                            <td>@t.DestinationWalletId</td>
+                            <td>@t.Amount</td>
+                            <td>@t.Date</td>
+                        </tr>
+                    }
+                </table>
+            }
+            else
+            {
+                <h6 class="card-subtitle mb-2 text-muted">You don't have any recived transaction...</h6>
+            }
+            <br />
+
+            <h5 class="card-title">Intern Transfers :</h5>
+            @if (InternTransactions.Count>0)
+            {
+                <table style="width:100%">
+                    <tr>
+                        <th>SourceWalletId</th>
+                        <th>DestinationWalletId</th>
+                        <th>Amount</th>
+                        <th>Date</th>
+                    </tr>
+                    @foreach (var t in InternTransactions)
+                    {
+                        <tr>
+                            <td>@t.SourceWalletId</td>
+                            <td>@t.DestinationWalletId</td>
+                            <td>@t.Amount</td>
+                            <td>@t.Date</td>
+                        </tr>  
+                    }
+                </table>
+            }
+            else
+            {
+                <h6 class="card-subtitle mb-2 text-muted">You don't have any intern transaction...</h6>
+            }
+        </div>
+    </div>
+}
+
+
+@code {
+    public List<Transaction> transaction_list { get; set; }
+    public List<Wallet> wallets { get; set; }
+    public List<Transaction> MadeTransactions = new List<Transaction>();
+    public List<Transaction> RecivedTransactions = new List<Transaction>();
+    public List<Transaction> InternTransactions = new List<Transaction>();
+
+    protected override async Task OnInitializedAsync()
+    {
+        try
+        {
+            transaction_list = await HttpClient.GetFromJsonAsync<List<Transaction>>($"api/wallet/transactions");
```

Comment:
you would want to limit the amount of transactions loaded on ui (what if there are thousands?). For example we want n=10 per page. Filtering and sorting should be done on backend, preferably in db (Because you can't sort if you don't have all transactions), and UI should receive n transactions.